learn/src/main/res/values/dimens.xml

```diff
@@ -29,4 +29,5 @@
     <dimen name="dimen_100_dp">100dp</dimen>
     <dimen name="dimen_slider_layout">100dp</dimen>
     <dimen name="dimen_youtube_view_height">250dp</dimen>
+    <dimen name="dimen_80_dp">80dp</dimen>
```

Comment:
creating dimen with value as its name isn't useful, its' the same as using hardcoded values. Please create dimens with meaningful name and if they can not be reused anywhere then they should be hardcoded